### PR TITLE
Add xyz.parlatype.Parlatype

### PR DESCRIPTION
**Pull request by gkarsay**
This application was named org.parlatype.Parlatype before. The last version with the old name was 4.1, this is version 4.2 with the new name.

### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly: *Media player specialised for transcribing speech; please note that this only a rename of org.parlatype.Parlatype. I did [end-of-life-rebase](https://github.com/flathub/org.parlatype.Parlatype/commit/c1275d73a2311031ee9ba970efa9be1613a0f009) before and published it (not sure I did it right because the app is now not available on Flathub anymore)*
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(Assuming `--filesystem=home:ro` is included in the minimal set.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer of the project. **Link: www.parlatype.xyz**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(There are no additional patches or files.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: xyz.parlatype.Parlatype.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/cmusphinx/sphinxbase/archive/9e901850ee4cd1c0b22f2d3b3c15f162564ca283.zip",
+                    "sha256": "4369fc4416b2bd3a396ca791cee1ca62c7360f549a78a3c754bb49ba0ebc843a"
+                }
+            ],
+        "cleanup": [
+            "/share/sphinxbase"
+            ]
```
Comment (bbhtt) [inline xyz.parlatype.Parlatype.json:43]: Indentation is wrong here

File: xyz.parlatype.Parlatype.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/cmusphinx/pocketsphinx/archive/ab6d6471800966990e12fdb6ed27ae36323cf2c4.zip",
+                    "sha256": "29ce0c81eda60bef919d42287bcf85d29b8dcb8dcb8dc69998f58a79a6e4a5c0"
+                }
+            ],
+        "cleanup": [
+            "/share/pocketsphinx"
+            ]
```
Comment (bbhtt) [inline xyz.parlatype.Parlatype.json:59]: here too

File: xyz.parlatype.Parlatype.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    "command": "parlatype",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--talk-name=org.freedesktop.secrets",
+        "--own-name=org.mpris.MediaPlayer2.parlatype"
```
Comment (bbhtt) [inline xyz.parlatype.Parlatype.json:17]: I'm pretty sure you need `org.mpris.MediaPlayer2.parlatype.*`

File: xyz.parlatype.Parlatype.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+    "command": "parlatype",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--talk-name=org.freedesktop.secrets",
```
Comment (bbhtt) [inline xyz.parlatype.Parlatype.json:16]: Why is this required? Does it support the secrets portal?

**Comment by bbhtt** [comment]
bot, build xyz.parlatype.Parlatype

File: xyz.parlatype.Parlatype.json (hunk 1 above)
Comment (gkarsay) [reply xyz.parlatype.Parlatype.json]: Fixed in https://github.com/flathub/flathub/pull/5312/commits/2be6165bb6284ca4562210d711c88b27cd6eff66

File: xyz.parlatype.Parlatype.json (hunk 2 above)
Comment (gkarsay) [reply xyz.parlatype.Parlatype.json]: Fixed in https://github.com/flathub/flathub/pull/5312/commits/2be6165bb6284ca4562210d711c88b27cd6eff66

File: xyz.parlatype.Parlatype.json (hunk 3 above)
Comment (gkarsay) [reply xyz.parlatype.Parlatype.json]: Done in https://github.com/flathub/flathub/pull/5312/commits/e2be8e385929578ce11968166683c564b88bfc73. Though it worked for me before, now too. I was like in org.gnome.Rhythmbox3 before.

File: xyz.parlatype.Parlatype.json (hunk 4 above)
Comment (gkarsay) [reply xyz.parlatype.Parlatype.json]: Removed in https://github.com/flathub/flathub/pull/5312/commits/0fc1e2a948614cea5466b7bcb939a08b08d44fa1

**Comment by gkarsay** [comment]
I added also `--device=dri` to get rid of some runtime warnings.

**Comment by bbhtt** [comment]
bot, build xyz.parlatype.Parlatype

**Comment by gkarsay** [comment]
@bbhtt Thanks for the review
